cucumber/api/features/step_definitions/audit_steps.rb

```diff
@@ -1,31 +1,37 @@
 # frozen_string_literal: true
 
+require_relative '../support/logs_helpers'
+
+Given(/^I save my place in the audit log file for remote$/) do
+  save_num_log_lines unless Utils.local_conjur_server
```

Comment:
What are we expecting to get when there is not local_conjur_server? the step will be empty? is it a possible scenario? Maybe we want to throw error?

Reply:
What do we think about cleaning audit file after each test instead of saving a pointer?

Reply:
If we already have `save_num_log_lines` implemented and use in other places, it's not necessary

Reply:
When there is local_conjur_server the step will not do anything, because the local server mechanism doesn't need to save any place in any log for it to work. It's not an error case.

Reply:
The log mechanism intentionally preferred not to delete the log with the following reason: https://github.com/cyberark/conjur/blob/8f4f9556222bdcbb896a03633b77b75d67b92533/cucumber/api/features/support/logs_helpers.rb#L5  . So I won't undermine their plans

Reply:
`save_num_log_lines` is indeed (almost) just a wrapper to a function defined in another place, but I added it here as well because it pairs with the step `there is an audit record matching` so I thought it's more readable to have them next to each other.